### PR TITLE
Set default value of LTSRelease to false

### PR DESCRIPTION
**Pull request by adityapatwardhan**
# PR Summary

Add the new value in `metadata.json` for LTS release and set it to false.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 1 Hour

File: tools/metadata.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
   "PreviewReleaseTag": "v7.0.0-rc.2",
```
Comment (PoshChan) [inline tools/metadata.json]: @adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-macos`
Verify sub-directory creation under root.Can create a sub directory under root path
```powershell
Expected path '\TestDir-2fc5afc7-5280-48d0-ad30-ef513711c201' to exist, but it did not exist.
at <ScriptBlock>, /Users/runner/runners/2.164.8/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 1467
1467:         $dirPath | Should -Exist
```

**Review by daxian-dbw — APPROVED** [review]

**Comment by adityapatwardhan** [comment]
@PoshChan please retry macos

**Comment by PoshChan** [comment]
@adityapatwardhan, successfully started retry of `PowerShell-CI-macOS`

File: tools/metadata.json (hunk 1 above)
Comment (PoshChan) [inline tools/metadata.json]: @adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-macos`
Verify sub-directory creation under root.Can create a sub directory under root path
```powershell
Expected path '\TestDir-19d3f162-077a-48ed-942c-8a9baf03a45d' to exist, but it did not exist.
at <ScriptBlock>, /Users/runner/runners/2.164.8/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 1467
1467:         $dirPath | Should -Exist
```

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 1 Hour ago

**Comment by adityapatwardhan** [comment]
@PoshChan please rebuild all

**Comment by PoshChan** [comment]
@adityapatwardhan, successfully started rebuild of `PowerShell-CI-static-analysis, PowerShell-CI-Windows, PowerShell-CI-macOS, PowerShell-CI-Linux`

**Pull request merged**